app-shell/src/config/index.ts

```diff
@@ -126,6 +148,7 @@ export function registerPythonPath(): Dispatch {
       case Cfg.OPEN_PYTHON_DIRECTORY: {
         const dir = getFullConfig().python.pathToPythonOverride
         if (dir != null) {
+          // eslint-disable-next-line @typescript-eslint/no-floating-promises
```

Comment:
not sure this is a good way to remove eslint-disable
```suggestion
         shell.openPath(dir).catch(err => {
            log().debug('Error opening python directory', err.message)
          })
```